doc/devdocs/object.rst

```diff
@@ -0,0 +1,88 @@
+******************************
```

Comment:
@vtjnash reminder to self: correct this section in https://github.com/JuliaLang/julia/pull/7906